packages/squiggle-lang/__tests__/library/distOperations_test.ts

```diff
@@ -0,0 +1,14 @@
+import { testEvalToBe } from "../helpers/reducerHelpers.js";
+
+// TODO - it'd be useful to do most of `./sym_test.ts` tests here, without `Sym.` prefix
+describe("Dist operations", () => {
+  describe("Power on negative samples", () => {
+    // ok when power is integer
+    testEvalToBe("normal(-100, 1) ^ 2", "Sample Set Distribution");
+    // fails when power is not an integer
+    testEvalToBe(
```

Comment:
Do you know why this fails when it's not an integer? It works fine in the JS console. (```3 ** 2.3```)

Reply:
It fails when base is negative, which shouldn't be surprising; consider `(-1) ^ 0.5`.

Reply:
Ah, makes sense